defaults/main.yml

```diff
@@ -19,6 +19,8 @@ redis_nofile_limit: 16384
 
 ## Role options
 # Configure Redis as a service
+redis_server: true
```

Comment:
it looks like the line is in the middle of a comment, maybe move it just below `## Role options` ?